src/launchpad/insights/insight.py

```diff
@@ -1,22 +1,11 @@
 from abc import abstractmethod
-from dataclasses import dataclass
 from typing import Protocol, TypeVar
 
-from ..models.apple import AppleAppInfo, MachOBinaryAnalysis
-from ..models.common import FileAnalysis
-from ..models.treemap import TreemapResults
+from launchpad.insights.common import InsightsInput
 
 T_co = TypeVar("T_co", covariant=True)
 
 
-@dataclass
```

Comment:
This class was duplicated. This version was iOS specific and the other was more generic so I removed this one.